src/BoundaryConditions/fill_halo_regions_open.jl

```diff
@@ -17,13 +17,19 @@ function fill_open_boundary_regions!(field, boundary_conditions, indices, loc, g
     open_fill, regular_fill = get_open_halo_filling_functions(loc) 
```

Comment:
This is a little convoluted. Why do you need a function to determine the size? Why can't we just use `loc`?

Reply:
This comes from normal halo filling and I think its main purpose is for windowed fields:
https://github.com/CliMA/Oceananigans.jl/blob/4dbec269c7fa88a611121a78924589945b535d2d/src/BoundaryConditions/fill_halo_regions.jl#L349-L433